migrator/src/test/java/com/google/phonenumbers/migrator/MetadataReaderTest.java

```diff
@@ -0,0 +1,14 @@
+package com.google.phonenumbers.migrator;
+
+import java.io.IOException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MetadataReaderTest {
+
+  @Test
+  public void testInvalidFileLocation() {
+    String fileLocation = "invalid-zipfile-location";
+    Assertions.assertThrows(IOException.class, () -> new MetadataReader(fileLocation));
```

Comment:
It sucks that JUnit4 doesn't have this method in it directly. There's one in:
  //src/test/java/com/google/i18n/phonenumbers/metadata/testing/AssertUtil.java
which you probably need to copy (since it's bad form to rely on another projects test code).

You can also assign the result of this to variable and make assertions about the error message (which is generally better because it lowers the risk of tests which *pass* when they shouldn't - which is a real pain).